lib/segment/src/vector_storage/quantized/quantized_multivector_storage.rs

```diff
@@ -20,6 +20,10 @@ pub struct MultivectorOffset {
     pub count: PointOffsetType,
 }
 
+pub trait MultivectorOffsets {
```

Comment:
A new trait is necessary to access the multivector offset from the query scorer.